GoMain/src/main/main.go

```diff
@@ -21,9 +21,9 @@ package main
 import (
 	"errors"
 	"log"
-	"time"
 
 	"common/logmgr"
+	"common/sigmgr"
```

Comment:
@suresh-lc We all recognize that `configuremgr` provides `watcher` to notify any state change of services on top of `edge-home-orchestration-go`. But this is for service notification. This PR suggests a separate manager called `sigmgr` to deal with Termination scenario of the edge-home-orchestration with respect to the newly suggested `SIGTERM`. What do you think of this approach from architecture perspective. IMHO, I think that this approach is quite valid.

Reply:
@t25kim : So the changes suggested here is for abnormal termination of edge-orchestration-go itself. From testing point, i understand it has edge-orchestration is started as docker container and stopped abruptly. So the abnormal stop gets handled. In Such case is the port also gets closed, can you check that once if not checked.

Reply:
@suresh-lc Thank you for the comment. This sigmgr could apply both of normal and abnormal termination scenarios.
For the normal example, if a user, who wants to terminate a edge container, run "docker stop edge-orchestration" and docker will produce SIGTERM and SIGKILL to stop it gracefully. [[REF]](https://docs.docker.com/engine/reference/commandline/stop/)

In this PR, I have checked the edge-orchestration goes to exit status when SIGTERM, SIGINT signals reach to the edge-orchestration process.

- Case 1. $ docker stop edge-orchestration
```
t25kim@t25kim:~$ docker ps
CONTAINER ID        IMAGE                       COMMAND             CREATED             STATUS              PORTS               NAMES
174666e6fe34        edge-orchestration:baobab   "sh run.sh"         5 seconds ago       Up 4 seconds                            edge-orchestration
t25kim@t25kim:~$ docker stop edge-orchestration
edge-orchestration
t25kim@t25kim:~$ docker ps -a
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                      PORTS               NAMES
174666e6fe34        edge-orchestration:baobab   "sh run.sh"              39 seconds ago      Exited (0) 8 seconds ago                        edge-orchestration
```
- Case 2.  "Ctrl + c" at the edge-orchestration process in the container
```
t25kim@t25kim:~$ docker ps
CONTAINER ID        IMAGE                       COMMAND             CREATED             STATUS              PORTS               NAMES
552e88e345a7        edge-orchestration:baobab   "sh run.sh"         13 seconds ago      Up 13 seconds                           edge-orchestration
t25kim@t25kim:~$ docker ps -a
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                       PORTS               NAMES
552e88e345a7        edge-orchestration:baobab   "sh run.sh"              31 seconds ago      Exited (130) 5 seconds ago                       edge-orchestration
```

- Case 3. SIGTERM to the edge-orchestration process in the container
```
t25kim@t25kim:~$ docker ps
CONTAINER ID        IMAGE                       COMMAND             CREATED             STATUS              PORTS               NAMES
ae2c43b75afe        edge-orchestration:baobab   "sh run.sh"         6 seconds ago       Up 5 seconds                            edge-orchestration
t25kim@t25kim:~$ docker exec -it edge-orchestration /bin/bash
root@t25kim:/edge-orchestration# ps -ef
UID        PID  PPID  C STIME TTY          TIME CMD
root         1     0  0 02:08 pts/0    00:00:00 sh run.sh
root         6     1  0 02:08 pts/0    00:00:00 /edge-orchestration/edge-orchestration
root        18     0  0 02:09 pts/1    00:00:00 /bin/bash
root        27    18  0 02:09 pts/1    00:00:00 ps -ef
root@t25kim:/edge-orchestration# kill -15 6
root@t25kim:/edge-orchestration# 
t25kim@t25kim:~$ docker ps -a
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                      PORTS               NAMES
ae2c43b75afe        edge-orchestration:baobab   "sh run.sh"              37 seconds ago      Exited (0) 15 seconds ago                       edge-orchestration
```

Reply:
@suresh-lc LGTM to accept this PR.